lib/WP_Auth0_LoginManager.php

```diff
@@ -232,9 +232,13 @@ public function redirect_login() {
 			throw new WP_Auth0_LoginFlowValidationException( $e_message, $e_code );
 		}
 
+		$access_token = $data->access_token;
+		$id_token = $data->id_token;
+		$refresh_token = isset( $data->refresh_token ) ? $data->refresh_token : null;
+
 		// Decode the incoming ID token for the Auth0 user.
 		$decoded_token = JWT::decode(
```

Comment:
does this work / would it make sense for this to be called if the token is missing?

Reply:
What do you mean by _both_? decode function is only using id_token.

Reply:
Sorry ... both `access_token` and `id_token`. They're both returned at the same time so if we have an `access_token`, we also have an `id_token`.